docs/blocks/block-events.md

```diff
@@ -295,10 +302,15 @@ Sets a block at a specified position relative to the block.
 <CodeHeader>minecraft:block > events</CodeHeader>
 
 ```json
-"wiki:generate_stone_above": {
```

Comment:
Please make this a separate codeblock and subheader. That way we can have a normal set_block and one with block states